core/src/main/protobuf/org/apache/spark/status/protobuf/store_types.proto

```diff
@@ -405,19 +405,19 @@ message SQLPlanMetric {
 
 message SQLExecutionUIData {
   int64 execution_id = 1;
-  string description = 2;
-  string details = 3;
-  string physical_plan_description = 4;
-  map<string, string> modified_configs = 5;
-  repeated SQLPlanMetric metrics = 6;
-  int64 submission_time = 7;
-  optional int64 completion_time = 8;
-  optional string error_message = 9;
-  map<int64, JobExecutionStatus> jobs = 10;
-  repeated int64 stages = 11;
-  bool metric_values_is_null = 12;
-  map<int64, string> metric_values = 13;
-  optional int64 root_execution_id = 14;
+  int64 root_execution_id = 2;
```

Comment:
since we haven't released the protobuf file, I move `root_execution_id` to the 2nd field and make it required instead of optional. This matches the definition of `SQLExecutionUIData`.